src/cephadm/cephadm.py

```diff
@@ -9454,7 +9457,10 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--ssh-user',
         default='root',
-        help='set user for SSHing to cluster hosts, passwordless sudo will be needed for non-root users')
+        help='set user for SSHing to cluster hosts, sudo will be needed for non-root users')
+    parser_bootstrap.add_argument(
+        '--ssh-password',
```

Comment:
Would think this should really be `--sudo-password`. Either way, we still have the issue of this requiring users to pass the password on the cmd line. That might be okay in certain setups where users are confident nobody will access the machine this is run on, but we should definitely also have some way of passing this through a file. We do this already for the `registry-login` command, which takes either username/password/repo as individual cmd line args, or takes a file that contains all the values. So you could look at how we handle that for an example.

Reply:
sudo-password part is done